Dockerfile

```diff
@@ -1,10 +1,25 @@
 FROM node:14-slim
+
+WORKDIR /usr/src/app
+
+RUN apt-get update && apt-get install -y \
+    python3 \
+    gcc \
+    build-essential \
+    && apt-get clean
+
 COPY package.json package.json
 COPY package-lock.json package-lock.json
+
 RUN npm install
```

Comment:
You could replace this with `npm install --production` and avoid installing dev dependencies 🤷